Main.java

```diff
@@ -0,0 +1,63 @@
+import java.util.*;
+// Main Class
+class Main
+{
+    static Scanner reader = new Scanner(System.in);
+    public static void main(String[] args)
+    {
+       Table table = new Table(3);
+        table.InsertColumnNames("name" , "last_name");
+        table.InsertRow("israel" , "israeli");
+        System.out.println( table.getValue("last_name", 0));
+    }
+}
+
+// Table Class
```

Comment:
THe guideline in comments is to add relevant information, or textually describe what a class does. In here you use the same words and give no new information